config/environments/production.rb

```diff
@@ -70,25 +80,32 @@
 
   # Enable locale fallbacks for I18n (makes lookups for any locale fall back to
   # the I18n.default_locale when a translation cannot be found).
-  config.i18n.fallbacks = {:tl => [:fil,I18n.default_locale]}
-  config.i18n.fallbacks.default = I18n.default_locale
+  config.i18n.fallbacks = true
```

Comment:
One thing we do is map the Tagalog "tl" locale/language translations as a fallback locale for the Filipino "fl" locale in production.

<details><summary>Context (click to expand):</summary>

There is considerable overlap in what is meant when discussing "Tagalog" / "Filipino" language, (https://en.wikipedia.org/wiki/Tagalog_language) so the locale files we have for Tagalog "tl" are shared/re-used to provide the Filipino "fl" locale.

This is done mostly to accommodate Firefox + Chrome/ium, with Firefox supporting the "tl" locale code and Chromium supporting the "fil" locale code.

</details>

I don't know if the existing syntax from before works (reverting these lines of diff?) without testing it myself, but I wanted to point this out to make sure I don't forget to address it at some point during the review.

If we can restore the "fil" --> "tl" fallback behavior, that would be great! Would like to see that restored before merging this.

Thanks again for working on this! It's much appreciated.

Reply:
Good to know, and that one must have slipped past as I tried to revert most of the things they changed automatically back to what they were before. I'll get this one reverted back!

Reply:
Pushed the fix for this :+1:

Reply:
The fix is working, thank you!

<details><summary>Testing details (click to expand):</summary>

Note that to test the `tl` --> `fil` locale fallback locally, one needs to copy the fallback logic (lines with `i18n` in them) from `config/environments/production.rb` to `config/environments/development.rb` so it is active in dev mode.

Also, one must go into their browser's preferences and set "tl" locale as the most-preferred language for web page content. (Might only be available in Firefox?)

I do see the `tl` --> `fil` fallback behavior working as intended, once the fallback config is applied to the dev environment.

(I don't super clearly recall why I didn't just set this config for `development` mode in the first place. Something about wanting to be reminded how the fallback would work if the `tl` locale is declared (as it is in [`config/application.rb`](https://github.com/RefugeRestrooms/refugerestrooms/blob/8a2449c43c6374a9a9830b4f071bd89442266edc/config/application.rb#L27)) but the fallback to `fil` isn't enabled. But maybe it's just harder to test, so I am considering copy-pasting the locale fallback logic into `config/environments/development.rb` as well. But I am not requesting any changes to that effect for this PR, just noting this down in case anyone is wondering how it can be set up and tested at the moment.)

</details>